### ISSUE TITLE
Inkscape does not do anything in aarch64-darwin.

### ISSUE THREAD

**Issue by PartemImperium** [NixOS/nixpkgs#153444]
### Describe the bug
When launching inkscape from command line the executable is started and it shows up in command + tab but no window shows up and wont quit unless force quit. If I install it as intel darwin (x86_64-darwin) either on an actual intel mac or through rosetta on a m1 it works just fine.

### Steps To Reproduce
1. Add inkscape to home-manager or any method of building inkscape
2. Run inkscape in the command line ("inkscape")

### Expected behavior
The app opens and is functional

### Screenshots
![image](https://user-images.githubusercontent.com/88595311/148021913-bc0f8171-7859-4805-b1d1-8d29199b4baa.png)

### Additional context
There is not any errors in the console (that are not also there when running the x86 version that works) but would be happy to share any log files that exist. I just dont know where they are to include here.

### Notify maintainers
@jtojnar 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"aarch64-darwin"`
 - host os: `Darwin 21.1.0, macOS 12.0.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.4`
 - channels(myUser): `"darwin, home-manager, nixpkgs-22.05pre340506.5c37ad87222"`
 - channels(root): `"darwin, home-manager, nixpkgs-22.05pre335173.56cbe42f166"`
 - nixpkgs: `/Users/myUser/.nix-defexpr/channels/nixpkgs`
```

**Comment by PartemImperium** [comment]
I noticed that even in x86 darwin the window that opens at first cant be quit (working as expected). I did an override to install it as x86 (on my m1 so it runs in rosetta). When opened I hit the checkmark to not open that window on app start. After that I went back to the m1 native version and it opened a blank document and the whole window is solid black. Quiting with cmd + q works now and when opened it goes to a new document again. Still with a solid black screen. 

I dont know what would cause the GUI to fail in this way but it may help identify the issue.
